packages/web/src/components/organisms/TransactionForm/index.tsx

```diff
@@ -76,9 +85,8 @@ export const TransactionForm = ({ propertyAddress }: Props) => {
   )
 
   useEffectAsync(async () => {
-    checkWithdrawable().then(withdrawable => {
-      setIsCancelCompleted(withdrawable)
-    })
+    const { withdrawable } = await checkWithdrawable()
```

Comment:
I refactored to prevent the nest from getting too deep.

Reply:
Thanks!!